II.write-tests/01-greet-people/greet-people.js

```diff
@@ -1,11 +1,11 @@
 function greetPeople(people) {
   var greeting = "Hello ";
-
-  people.forEach(function(person) {
-    greeting = greeting + person;
+  var myarray = []
+  people.forEach(function (person) {
+    myarray.push(greeting + person);
   });
 
-  return greeting;
+  return myarray;
```

Comment:
nice work, while forEach is pushing to another array (not quite its role)
something like map will help you better